server/src/main/java/com/tencent/rss/server/ShuffleServerGrpcService.java

```diff
@@ -225,19 +231,23 @@ public void finishShuffle(FinishShuffleRequest req,
     int shuffleId = req.getShuffleId();
     StatusCode status;
     String msg = "OK";
-    String errorMsg = "Fail to finish shuffle for appId["
-        + appId + "], shuffleId[" + shuffleId + "], data may be lost";
-    try {
-      LOG.info("Get finishShuffle request for appId[" + appId + "], shuffleId[" + shuffleId + "]");
-      status = shuffleServer.getShuffleTaskManager().commitShuffle(appId, shuffleId);
-      if (status != StatusCode.SUCCESS) {
+    if (shuffleServer.isMemoryShuffleEnabled()) {
```

Comment:
Memory Client not commit, Server can also process normal client.

Reply:
ok,  client should decide if send commit rpc to shuffle server